packages/web/src/components/ui/BarChart/BarChart.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useMemo } from 'react';
+import styles from './BarChart.module.scss';
+import clsx from 'clsx';
+import { formatNumber } from '../../../utils/helpers';
+import { Icon } from '../Icon/Icon';
+
+type Bar = {
+  fill: string;
```

Comment:
We'll probably want this to be a floating-point number ranging 0 to 1, rather than a string